diff -Nur selinux-basics-0.5.8/debian/control selinux-basics-0.5.8-0gardenlinux1/debian/control
--- selinux-basics-0.5.8/debian/control	2020-12-31 03:10:42.000000000 +0000
+++ selinux-basics-0.5.8-0gardenlinux1/debian/control	2021-08-08 14:24:21.762932966 +0000
@@ -16,7 +16,6 @@
          policycoreutils (>= 3.1),
          selinux-utils (>= 3.1),
          ${misc:Depends},
-         ${perl:Depends},
          ${python3:Depends}
 Recommends: policycoreutils-python-utils (>= 3.1),
             selinux-policy-default,
@@ -27,3 +26,15 @@
 Description: SELinux basic support
  This package will pull in basic SELinux stuff to ease installation, as well
  as provide scripts and helpers to work around common problems.
+
+Package: selinux-postfix
+Architecture: all
+Depends: selinux-basics,
+         ${misc:Depends},
+         ${perl:Depends},
+         ${python3:Depends}
+Recommends: postfix,
+Pre-Depends: ${misc:Pre-Depends}
+Description: SELinux support for postfix
+ This package will pull in basic SELinux and add a postfix rule, to ease
+ support for postfix in SELinux.
diff -Nur selinux-basics-0.5.8/debian/selinux-basics.install selinux-basics-0.5.8-0gardenlinux1/debian/selinux-basics.install
--- selinux-basics-0.5.8/debian/selinux-basics.install	2020-01-04 12:02:00.000000000 +0000
+++ selinux-basics-0.5.8-0gardenlinux1/debian/selinux-basics.install	2021-08-08 14:24:54.082933668 +0000
@@ -1,5 +1,4 @@
 check-selinux-installation /usr/sbin/
-postfix-nochroot /usr/sbin
 selinux-activate /usr/sbin
 selinux-config-enforcing /usr/sbin
 selinux-policy-upgrade /usr/sbin
diff -Nur selinux-basics-0.5.8/debian/selinux-basics.manpages selinux-basics-0.5.8-0gardenlinux1/debian/selinux-basics.manpages
--- selinux-basics-0.5.8/debian/selinux-basics.manpages	2020-01-04 12:02:00.000000000 +0000
+++ selinux-basics-0.5.8-0gardenlinux1/debian/selinux-basics.manpages	2021-08-08 14:26:09.678935311 +0000
@@ -1,4 +1,3 @@
 check-selinux-installation.8
-postfix-nochroot.8
 selinux-config-enforcing.8
 selinux-policy-upgrade.8
diff -Nur selinux-basics-0.5.8/debian/selinux-postfix.install selinux-basics-0.5.8-0gardenlinux1/debian/selinux-postfix.install
--- selinux-basics-0.5.8/debian/selinux-postfix.install	1970-01-01 00:00:00.000000000 +0000
+++ selinux-basics-0.5.8-0gardenlinux1/debian/selinux-postfix.install	2021-08-08 14:24:58.970933775 +0000
@@ -0,0 +1 @@
+postfix-nochroot /usr/sbin
diff -Nur selinux-basics-0.5.8/debian/selinux-postfix.manpages selinux-basics-0.5.8-0gardenlinux1/debian/selinux-postfix.manpages
--- selinux-basics-0.5.8/debian/selinux-postfix.manpages	1970-01-01 00:00:00.000000000 +0000
+++ selinux-basics-0.5.8-0gardenlinux1/debian/selinux-postfix.manpages	2021-08-08 14:25:25.326934347 +0000
@@ -0,0 +1 @@
+postfix-nochroot.8
