--- org/debian/patches/series	2021-07-20 06:52:01.208048015 +0000
+++ new/debian/patches/series	2021-07-20 06:53:31.031768019 +0000
@@ -94,9 +94,9 @@
 bugfix/all/partially-revert-usb-kconfig-using-select-for-usb_co.patch
 debian/makefile-do-not-check-for-libelf-when-building-oot-module.patch
 bugfix/all/partially-revert-net-socket-implement-64-bit-timestamps.patch
-bugfix/all/Revert-PCI-PM-Do-not-read-power-state-in-pci_enable_.patch
 
 # Miscellaneous features
+fpga-ofs.patch
 
 # Lockdown missing pieces
 features/all/lockdown/efi-add-an-efi_secure_boot-flag-to-indicate-secure-b.patch
