--- org/debian/config/config	2021-12-02 10:21:47.023443611 +0000
+++ new/debian/config/config	2021-12-02 10:22:33.832683889 +0000
@@ -190,9 +190,9 @@
 ## file: drivers/android/Kconfig
 ##
 # CONFIG_ANDROID is not set
-CONFIG_ANDROID_BINDER_IPC=m
+# CONFIG_ANDROID_BINDER_IPC=m
 #. This configuration should support Anbox
-CONFIG_ANDROID_BINDER_DEVICES="binder"
+# CONFIG_ANDROID_BINDER_DEVICES="binder"
 # CONFIG_ANDROID_BINDER_IPC_SELFTEST is not set
 
 ##
--- org/debian/config/amd64/config	2021-12-02 10:24:22.503563776 +0000
+++ new/debian/config/amd64/config	2021-12-02 10:24:34.883891907 +0000
@@ -91,7 +91,7 @@
 ##
 ## file: drivers/android/Kconfig
 ##
-CONFIG_ANDROID=y
+# CONFIG_ANDROID=y
 
 ##
 ## file: drivers/block/Kconfig
--- org/debian/config/arm64/config	2021-12-02 10:25:22.861163574 +0000
+++ new/debian/config/arm64/config	2021-12-02 10:25:54.173993595 +0000
@@ -91,7 +91,7 @@
 ##
 ## file: drivers/android/Kconfig
 ##
-CONFIG_ANDROID=y
+# CONFIG_ANDROID=y
 
 ##
 ## file: drivers/ata/Kconfig
--- org/debian/config/armhf/config	2021-12-02 10:25:35.769505735 +0000
+++ new/debian/config/armhf/config	2021-12-02 10:26:01.358184034 +0000
@@ -164,7 +164,7 @@
 ##
 ## file: drivers/android/Kconfig
 ##
-CONFIG_ANDROID=y
+# CONFIG_ANDROID=y
 
 ##
 ## file: drivers/ata/Kconfig
 