--- org/debian/installer/modules/scsi-modules	2021-11-18 16:56:25.570310172 +0000
+++ new/debian/installer/modules/scsi-modules	2021-11-18 16:56:34.030527712 +0000
@@ -27,7 +27,6 @@
 tcm_qla2xxx -
 
 # Exclude common code packaged in {cdrom,scsi}-core-modules
-scsi_common -
 scsi_mod -
 sd_mod -
 sr_mod -
--- org/debian/installer/modules/scsi-core-modules	2021-11-18 16:57:35.812118766 +0000
+++ new/debian/installer/modules/scsi-core-modules	2021-11-18 16:57:41.140256174 +0000
@@ -1,4 +1,3 @@
-scsi_common
 scsi_mod
 sd_mod
 scsi_transport_sas ?
