--- cyrus-sasl2-2.1.27+dfsg.orig/debian/control	2020-05-24 23:20:41.284611261 +0000
+++ cyrus-sasl2-2.1.27+dfsg/debian/control	2020-05-24 23:21:18.356744248 +0000
@@ -14,7 +14,7 @@
                groff-base,
                heimdal-multidev <!pkg.cyrus-sasl2.nogssapi>,
                krb5-multidev <!pkg.cyrus-sasl2.nogssapi>,
-               libdb-dev,
+               libgdbm-dev,
                libkrb5-dev <!pkg.cyrus-sasl2.nogssapi>,
                libldap2-dev <!pkg.cyrus-sasl2.noldap>,
                libpam0g-dev,
--- cyrus-sasl2-2.1.27+dfsg.orig/debian/rules	2020-05-24 23:20:11.524504532 +0000
+++ cyrus-sasl2-2.1.27+dfsg/debian/rules	2020-05-24 23:20:31.588576486 +0000
@@ -89,6 +89,7 @@
 	--enable-ntlm \
 	--disable-passdss \
 	$(CONFIGURE_SQL) \
+	--with-dblib=gdbm \
 	--with-sqlite3=/usr \
 	--with-mysql=/usr \
 	--with-pgsql=/usr \
