+++ iproute2-5.6.0/debian/control	2020-05-23 06:17:49.779719967 +0000
@@ -15,7 +15,6 @@
                libatm1-dev,
                libbsd-dev,
                libcap-dev,
-               libdb-dev,
                libelf-dev,
                libmnl-dev,
                libselinux1-dev,
@@ -27,8 +26,7 @@
 Package: iproute2
 Priority: important
 Architecture: linux-any
-Provides: arpd
-Conflicts: arpd, iproute (<< 20130000-1)
+Conflicts: iproute (<< 20130000-1)
 Replaces: iproute
 Depends: ${misc:Depends}, ${shlibs:Depends}, libcap2-bin,
 Recommends: ${ipmods:Recommends}
+++ iproute2-5.6.0/debian/iproute2.install	2020-05-23 06:19:13.248027814 +0000
@@ -1,6 +1,5 @@
 etc/
 usr/include/iproute2/
-sbin/arpd /usr/sbin
 sbin/bridge
 sbin/devlink
 sbin/genl /usr/sbin
