--- org/debian/control	2021-10-17 20:49:56.041293156 +0000
+++ new/debian/control	2021-10-17 20:51:27.827728704 +0000
@@ -2,12 +2,20 @@
 Maintainer: Google Cloud Team <gc-team@google.com>
 Section: misc
 Priority: optional
-Standards-Version: 3.9.8
-Build-Depends: debhelper (>= 9.20160709), dh-golang (>= 1.1), golang-go
+Testsuite: autopkgtest-pkg-go
+Build-Depends: debhelper-compat (= 12),
+               dh-golang,
+               golang-any
+Standards-Version: 4.5.0
+Homepage: https://github.com/GoogleCloudPlatform/guest-agent
+Rules-Requires-Root: no
+XS-Go-Import-Path: github.com/GoogleCloudPlatform/guest-agent
 
 Package: google-guest-agent
 Architecture: any
-Depends: ${misc:Depends}
+Depends: ${misc:Depends},
+         ${shlibs:Depends}
+Built-Using: ${misc:Built-Using}
 Conflicts: python-google-compute-engine, python3-google-compute-engine
 Description: Google Compute Engine Guest Agent
  Contains the guest agent and metadata script runner binaries.
--- org/debian/compat	2021-10-17 21:05:11.181586993 +0000
+++ new/debian/compat	1970-01-01 00:00:00.000000000 +0000
@@ -1 +0,0 @@
-10
--- org/debian/install	1970-01-01 00:00:00.000000000 +0000
+++ new/debian/install	2021-10-17 21:15:36.926204316 +0000
@@ -0,0 +1 @@
+*.service lib/systemd/system
--- org/debian/rules	2021-10-17 20:13:16.178767160 +0000
+++ new/debian/rules	2021-10-17 20:20:19.462070970 +0000
@@ -1,16 +1,7 @@
 #!/usr/bin/make -f
 
-export PATH := /tmp/go/bin:$(PATH)
-export SHELL := env PATH=$(PATH) /bin/bash
-
-export DH_OPTIONS
-export DH_GOPKG := github.com/GoogleCloudPlatform/guest-agent
-export CGO_ENABLED := 0
-export GOCACHE := /tmp/.cache
-export GO111MODULE := on
-
 %:
-	dh $@  --buildsystem=golang --with=golang,systemd
+	dh $@ --builddirectory=_build --buildsystem=golang --with=golang
 
 override_dh_auto_install:
 	# Binary-only package.
@@ -18,22 +9,16 @@
 	install -d debian/google-guest-agent/usr/share/google-guest-agent
 	install -p -m 0644 instance_configs.cfg debian/google-guest-agent/usr/share/google-guest-agent
 
-override_dh_golang:
-	# We don't use any packaged dependencies, so skip dh_golang step.
-
+override_dh_auto_configure:
+	cp -r debian/extra/vendor ./
+	dh_auto_configure
+
+override_dh_clean:
+	dh_clean vendor/
+ 
 override_dh_auto_build:
-	dh_auto_build -O--buildsystem=golang -- -ldflags="-s -w -X main.version=$(VERSION)" -mod=readonly
-
-override_dh_installinit:
-	# We don't ship sysvinit files or need script changes for them.
-
-override_dh_systemd_enable:
-	install -d debian/google-guest-agent/lib/systemd/system
-	install -p -m 0644 *.service debian/google-guest-agent/lib/systemd/system/
-	install -d debian/google-guest-agent/lib/systemd/system-preset
-	install -p -m 0644 *.preset debian/google-guest-agent/lib/systemd/system-preset/
-	dh_systemd_enable google-guest-agent.service google-startup-scripts.service google-shutdown-scripts.service
+	dh_auto_build -O--buildsystem=golang -- -ldflags="-s -w -X main.version=$(shell dpkg-parsechangelog -S Version)"
 
-override_dh_systemd_start:
-	# Only perform start/stop actions for the guest agent.
-	dh_systemd_start google-guest-agent.service
+override_dh_installsystemd:
+	dh_installsystemd google-guest-agent.service
+	dh_installsystemd google-startup-scripts.service google-shutdown-scripts.service
