Debian Bug: #930684, #968927
Commit: https://salsa.debian.org/installer-team/debootstrap/-/commit/87cdebbcad6f4e16ba711227cbbbd70039f88752
--- /usr/share/debootstrap/functions	2020-03-19 23:52:15.932784227 +0000
+++ /usr/share/debootstrap/functions	2020-03-19 23:53:56.137329010 +0000
@@ -1197,7 +1197,9 @@
 			rm -f $TARGET/proc
 			mkdir $TARGET/proc
 		else
-			umount "$TARGET/proc" 2>/dev/null || true
+			if [ "$CONTAINER" != "docker" ]; then
+				umount "$TARGET/proc" 2>/dev/null || true
+			fi
 		fi

 		# some container environment are used at second-stage, it already treats /proc and so on
