+++ python3.8/debian/control.in
@@ -15,7 +15,7 @@
   locales-all,
   libsqlite3-dev, libffi-dev (>= 3.0.5) [!or1k !avr32],
   libgpm2 [linux-any],
-  mime-support, netbase, bzip2, time, python3@bd_qual@, @PVER@@bd_qual@ <cross>,
+  media-types, netbase, bzip2, time, python3@bd_qual@, @PVER@@bd_qual@ <cross>,
   net-tools, xvfb <!nocheck>, xauth <!nocheck>,
   systemtap-sdt-dev
 Build-Depends-Indep: python3-sphinx, python3-docs-theme, texinfo
@@ -30,7 +30,7 @@
 Priority: @PRIO@
 Depends: @PVER@-minimal (= ${binary:Version}),
  lib@PVER@-stdlib (= ${binary:Version}),
- mime-support,
+ media-types,
  ${shlibs:Depends}, ${misc:Depends}
 Suggests: @PVER@-venv, @PVER@-doc, binutils
 Breaks: python3-all (<< 3.6.5~rc1-1),
@@ -62,7 +62,7 @@
 Pre-Depends: ${misc:Pre-Depends}
 Breaks: libpython3.8-testsuite (<< 3.8.0-4)
 Replaces: libpython3.8-testsuite (<< 3.8.0-4)
-Depends: lib@PVER@-minimal (= ${binary:Version}), mime-support, ${shlibs:Depends}, ${misc:Depends}
+Depends: lib@PVER@-minimal (= ${binary:Version}), media-types, ${shlibs:Depends}, ${misc:Depends}
 Description: Interactive high-level object-oriented language (standard library, version @VER@)
  Python is a high-level, interactive, object-oriented language. Its @VER@ version
  includes an extensive class library with lots of goodies for
