--- org/debian/patches/series	2021-07-20 06:52:01.208048015 +0000
+++ new/debian/patches/series	2021-07-20 06:53:31.031768019 +0000
@@ -76,7 +76,6 @@
 bugfix/powerpc/powerpc-boot-fix-missing-crc32poly.h-when-building-with-kernel_xz.patch
 bugfix/arm64/arm64-acpi-Add-fixup-for-HPE-m400-quirks.patch
 bugfix/x86/x86-32-disable-3dnow-in-generic-config.patch
-bugfix/x86/Revert-drm-i915-Implement-Wa_1508744258.patch

 # Arch features
 features/arm64/arm64-dts-rockchip-Add-support-for-two-PWM-fans-on-h.patch
@@ -87,8 +86,6 @@
 # Miscellaneous bug fixes
 bugfix/all/disable-some-marvell-phys.patch
 bugfix/all/fs-add-module_softdep-declarations-for-hard-coded-cr.patch
-bugfix/all/fuse-release-pipe-buf-after-last-use.patch
-bugfix/all/nfsd-fix-use-after-free-due-to-delegation-race.patch

 # Miscellaneous features

@@ -108,14 +105,6 @@
 # Security fixes
 debian/i386-686-pae-pci-set-pci-nobios-by-default.patch
 debian/ntfs-mark-it-as-broken.patch
-bugfix/all/atlantic-Fix-OOB-read-and-write-in-hw_atl_utils_fw_r.patch
-bugfix/all/fget-check-that-the-fd-still-exists-after-getting-a-.patch
-bugfix/all/USB-gadget-detect-too-big-endpoint-0-requests.patch
-bugfix/all/USB-gadget-zero-allocate-endpoint-0-buffers.patch
-bugfix/all/bpf-fix-kernel-address-leakage-in-atomic-fetch.patch
-bugfix/all/bpf-fix-signed-bounds-propagation-after-mov32.patch
-bugfix/all/bpf-make-32-64-bounds-propagation-slightly-more-robust.patch
-bugfix/all/bpf-fix-kernel-address-leakage-in-atomic-cmpxchg-s-r0-aux-reg.patch

 # Fix exported symbol versions
 bugfix/all/module-disable-matching-missing-version-crc.patch