--- org/debian/patches/series	2021-07-20 06:52:01.208048015 +0000
+++ new/debian/patches/series	2021-07-20 06:53:31.031768019 +0000
@@ -97,6 +97,7 @@ debian/makefile-do-not-check-for-libelf-
 bugfix/all/partially-revert-net-socket-implement-64-bit-timestamps.patch

 # Miscellaneous features
+garden/kernel/fpga-ofs.patch

 # Lockdown missing pieces
 features/all/lockdown/efi-add-an-efi_secure_boot-flag-to-indicate-secure-b.patch
@@ -106,9 +107,7 @@ features/all/lockdown/arm64-add-kernel-c

 # Improve integrity platform keyring for kernel modules verification
 features/all/db-mok-keyring/0001-MODSIGN-do-not-load-mok-when-secure-boot-disabled.patch
-features/all/db-mok-keyring/0002-MODSIGN-load-blacklist-from-MOKx.patch
 features/all/db-mok-keyring/0003-MODSIGN-checking-the-blacklisted-hash-before-loading-a-kernel-module.patch
-features/all/db-mok-keyring/0004-MODSIGN-check-the-attributes-of-db-and-mok.patch
 features/all/db-mok-keyring/modsign-make-shash-allocation-failure-fatal.patch
 features/all/db-mok-keyring/KEYS-Make-use-of-platform-keyring-for-module-signature.patch
