--- a/debian/dracut-core.install
+++ b/debian/dracut-core.install
@@ -3,6 +3,7 @@ usr/bin/dracut-catimages
 usr/bin/lsinitrd
 usr/lib/dracut/dracut-functions
 usr/lib/dracut/dracut-functions.sh
+usr/lib/dracut/dracut-util
 usr/lib/dracut/dracut-init.sh
 usr/lib/dracut/dracut-initramfs-restore
 usr/lib/dracut/dracut-install
@@ -70,3 +71,23 @@ modules.d/98usrmount		usr/lib/dracut/modules.d
 modules.d/99base		usr/lib/dracut/modules.d
 modules.d/99fs-lib		usr/lib/dracut/modules.d
 modules.d/99shutdown		usr/lib/dracut/modules.d
+modules.d/01systemd-ac-power    usr/lib/dracut/modules.d
+modules.d/01systemd-ask-password usr/lib/dracut/modules.d
+modules.d/01systemd-coredump 	usr/lib/dracut/modules.d
+modules.d/01systemd-hostnamed	usr/lib/dracut/modules.d
+modules.d/01systemd-journald	usr/lib/dracut/modules.d
+modules.d/01systemd-ldconfig	usr/lib/dracut/modules.d
+modules.d/01systemd-modules-load	usr/lib/dracut/modules.d
+modules.d/01systemd-repart	usr/lib/dracut/modules.d
+modules.d/01systemd-rfkill	usr/lib/dracut/modules.d
+modules.d/01systemd-sysctl	usr/lib/dracut/modules.d
+modules.d/01systemd-sysext	usr/lib/dracut/modules.d
+modules.d/01systemd-sysusers	usr/lib/dracut/modules.d
+modules.d/01systemd-timedated	usr/lib/dracut/modules.d
+modules.d/01systemd-tmpfiles	usr/lib/dracut/modules.d
+modules.d/01systemd-udevd	usr/lib/dracut/modules.d
+modules.d/01systemd-veritysetup	usr/lib/dracut/modules.d
+modules.d/06dbus-broker		usr/lib/dracut/modules.d
+modules.d/06dbus-daemon		usr/lib/dracut/modules.d
+modules.d/09dbus		usr/lib/dracut/modules.d
+modules.d/91tpm2-tss		usr/lib/dracut/modules.d
--- a/debian/dracut-network.install
+++ b/debian/dracut-network.install
@@ -20,3 +20,5 @@ modules.d/95nfs		usr/lib/dracut/modules.d
 modules.d/95ssh-client	usr/lib/dracut/modules.d
 modules.d/95virtfs	usr/lib/dracut/modules.d
 modules.d/99uefi-lib	usr/lib/dracut/modules.d
+modules.d/01systemd-resolved	usr/lib/dracut/modules.d
+modules.d/01systemd-timesyncd	usr/lib/dracut/modules.d
