--- a/modules.d/40network/module-setup.sh	2020-12-14 17:54:26.000000000 +0000
+++ b/modules.d/40network/module-setup.sh	2021-02-11 17:54:26.000000000 +0000
@@ -9,7 +9,7 @@
 depends() {
     is_qemu_virtualized && echo -n "qemu-net "
 
-    for module in network-wicked network-manager network-legacy ; do
+    for module in systemd-networkd network-wicked network-manager network-legacy ; do
         if dracut_module_included "$module" ; then
                 network_handler="$module"
                 break
