--- a/modules.d/02systemd-networkd/module-setup.sh	2020-01-30 15:32:19.000000000 +0000
+++ b/modules.d/02systemd-networkd/module-setup.sh	2020-02-12 15:32:19.000000000 +0000
@@ -26,9 +26,17 @@
     inst_multiple -o \
         $systemdutildir/systemd-networkd \
         $systemdutildir/systemd-networkd-wait-online \
+        $systemdutildir/systemd-network-generator \
+        $systemdutildir/systemd-resolved \
         $systemdsystemunitdir/systemd-networkd-wait-online.service \
         $systemdsystemunitdir/systemd-networkd.service \
+        $systemdsystemunitdir/systemd-network-generator.service \
+        $systemdsystemunitdir/systemd-resolved.service \
         $systemdsystemunitdir/systemd-networkd.socket \
+        $systemdsystemunitdir/network-online.target \
+        $systemdsystemunitdir/network-pre.target \
+        $systemdsystemunitdir/network.target \
+        $systemdsystemunitdir/systemd-resolved.service.d/resolvconf.conf \
         $systemdutildir/network/99-default.link \
         networkctl ip
 
@@ -50,9 +58,13 @@
 
     grep '^systemd-network:' $dracutsysrootdir/etc/passwd 2>/dev/null >> "$initdir/etc/passwd"
     grep '^systemd-network:' $dracutsysrootdir/etc/group >> "$initdir/etc/group"
+    grep '^systemd-resolve:' $dracutsysrootdir/etc/passwd 2>/dev/null >> "$initdir/etc/passwd"
+    grep '^systemd-resolve:' $dracutsysrootdir/etc/group >> "$initdir/etc/group"
     # grep '^systemd-timesync:' $dracutsysrootdir/etc/passwd 2>/dev/null >> "$initdir/etc/passwd"
     # grep '^systemd-timesync:' $dracutsysrootdir/etc/group >> "$initdir/etc/group"
 
+    ln -s /run/systemd/resolve/resolv.conf "$initdir/etc/resolv.conf" 
+
     _arch=${DRACUT_ARCH:-$(uname -m)}
     inst_libdir_file {"tls/$_arch/",tls/,"$_arch/",}"libnss_dns.so.*" \
                      {"tls/$_arch/",tls/,"$_arch/",}"libnss_mdns4_minimal.so.*" \
@@ -62,10 +74,14 @@
     for i in \
         systemd-networkd-wait-online.service \
         systemd-networkd.service \
-        systemd-networkd.socket
+        systemd-networkd.socket \
+        systemd-network-generator.service \
+        systemd-resolved.service \
 #       systemd-timesyncd.service
     do
         systemctl -q --root "$initdir" enable "$i"
     done
-}
 
+    # resolved - name resolution in dracut
+    systemctl -q --root "$initdir" add-wants network-online.target systemd-resolved.service
+}
