Original : https://github.com/dracutdevs/dracut/pull/1658
--- a/modules.d/01systemd-sysusers/module-setup.sh
+++ b/modules.d/01systemd-sysusers/module-setup.sh
@@ -24,6 +24,8 @@ depends() {
 # Install the required file(s) and directories for the module in the initramfs.
 install() {
 
+    inst_simple "$moddir/sysusers-dracut.conf" "$systemdsystemunitdir/systemd-sysusers.service.d/sysusers-dracut.conf"
+
     inst_multiple -o \
         "$sysusers"/basic.conf \
         "$sysusers"/systemd.conf \
--- /dev/null
+++ b/modules.d/01systemd-sysusers/sysusers-dracut.conf
@@ -0,0 +1,2 @@
+[Unit]
+ConditionNeedsUpdate=
