+++ pam-2.1.2/debian/control
@@ -15,7 +15,6 @@
                gettext (>= 0.12),
                googletest <!nocheck> | libgtest-dev <!nocheck>,
                libbz2-dev,
-               libdb-dev,
                libgnutls28-dev (>= 3.4.6),
                libgcrypt20-dev,
                liblz4-dev (>= 0.0~r126),
@@ -140,8 +139,6 @@
  .
   * apt-extracttemplates is used by debconf to prompt for configuration
     questions before installation.
-  * apt-ftparchive is used to create Packages and other index files
-    needed to publish an archive of Debian packages
   * apt-sortpkgs is a Packages/Sources file normalizer.
 
 Package: apt-transport-https
+++ pam-2.1.2/debian/apt-utils.install
@@ -1,13 +1,10 @@
 usr/bin/apt-extracttemplates
-usr/bin/apt-ftparchive
 usr/bin/apt-sortpkgs
 usr/lib/apt/planners/
 usr/lib/apt/solvers/
 usr/share/doc/apt-utils
 usr/share/locale/*/*/apt-utils.mo
 usr/share/man/*/*/apt-extracttemplates.*
-usr/share/man/*/*/apt-ftparchive.*
 usr/share/man/*/*/apt-sortpkgs.*
 usr/share/man/*/apt-extracttemplates.*
-usr/share/man/*/apt-ftparchive.*
 usr/share/man/*/apt-sortpkgs.*
