+++ perl/debian/control	
@@ -8,7 +8,6 @@
 Rules-Requires-Root: no
 Build-Depends: file,
  cpio,
- libdb-dev,
  libgdbm-dev (>= 1.18-3),
  libgdbm-compat-dev,
  netbase <!nocheck>,
+++ perl/debian/config.over
@@ -33,10 +33,10 @@
 fi

 # we always want the Berkeley DB support
-if [ "$i_db" != "$define" ]; then
-    echo "debian/config.over: Berkeley DB support not enabled, aborting!" 1>&2
-    exit 1
-fi
+#if [ "$i_db" != "$define" ]; then
+#    echo "debian/config.over: Berkeley DB support not enabled, aborting!" 1>&2
+#    exit 1
+#fi

 # force /usr/lib/<triplet> into $Config{libpth} if it's not there yet
 # see #630399
