+++ python3.8-3.8.3/debian/rules
@@ -369,7 +369,7 @@
 		--prefix=/usr \
 		--enable-ipv6 \
 		--enable-loadable-sqlite-extensions \
-		--with-dbmliborder=bdb:gdbm \
+		--with-dbmliborder=gdbm \
 		--with-computed-gotos \
 		--without-ensurepip \
 		--with-system-expat \
+++ python3.8-3.8.3/debian/control	
@@ -7,7 +7,7 @@
   lsb-release, sharutils,
   libreadline-dev, libncursesw5-dev (>= 5.3), 
   zlib1g-dev, libbz2-dev, liblzma-dev,
-  libgdbm-dev, libdb-dev,
+  libgdbm-dev,
   tk-dev, blt-dev (>= 2.4z), libssl-dev,
   libexpat1-dev, libmpdec-dev (>= 2.4),
   libbluetooth-dev [linux-any] <!pkg.python3.8.nobluetooth>,
