+++ python3.8/debian/control.in
@@ -7,14 +7,14 @@
   lsb-release, sharutils,
   libreadline-dev, libncursesw5-dev (>= 5.3), @bd_gcc@
   zlib1g-dev, libbz2-dev, liblzma-dev,
-  libgdbm-dev, libdb-dev,
+  libgdbm-dev,
   tk-dev, blt-dev (>= 2.4z), libssl-dev,
   libexpat1-dev, libmpdec-dev (>= 2.4),
   libbluetooth-dev [linux-any] <!pkg.@PVER@.nobluetooth>,
   locales-all,
   libsqlite3-dev, libffi-dev (>= 3.0.5) [!or1k !avr32],
   libgpm2 [linux-any],
-  mime-support, netbase, bzip2, time, python3@bd_qual@, @PVER@@bd_qual@ <cross>,
+  media-types, netbase, bzip2, time, python3@bd_qual@, @PVER@@bd_qual@ <cross>,
   net-tools, xvfb <!nocheck>, xauth <!nocheck>,
   systemtap-sdt-dev
 Build-Depends-Indep: python3-sphinx, python3-docs-theme, texinfo
@@ -30,7 +30,7 @@
 Priority: @PRIO@
 Depends: @PVER@-minimal (= ${binary:Version}),
  lib@PVER@-stdlib (= ${binary:Version}),
- mime-support,
+ media-types,
  ${shlibs:Depends}, ${misc:Depends}
 Suggests: @PVER@-venv, @PVER@-doc, binutils
 Breaks: python3-all (<< 3.6.5~rc1-1),
+++ python3.8/debian/rules
@@ -373,7 +373,7 @@
 		--prefix=/usr \
 		--enable-ipv6 \
 		--enable-loadable-sqlite-extensions \
-		--with-dbmliborder=bdb:gdbm \
+		--with-dbmliborder=gdbm \
 		--with-computed-gotos \
 		--without-ensurepip \
 		--with-system-expat \
+++ python3.9/debian/tests/control
@@ -2,7 +2,7 @@
 Depends: build-essential,
   locales-all,
   python3.9-dev,
-  libpython3.9-testsuite:native,
+  libpython3.9-testsuite,
   python3-gdbm,
   python3-distutils,
   python3-tk,
@@ -15,7 +15,7 @@
 Depends: build-essential,
   locales-all, python3.9-dev,
   python3.9-dbg,
-  libpython3.9-testsuite:native,
+  libpython3.9-testsuite,
   python3-gdbm-dbg,
   gdb,
   python3-distutils,
@@ -29,7 +29,7 @@
 Depends: build-essential,
   locales-all,
   python3.9-dev,
-  libpython3.9-testsuite:native,
+  libpython3.9-testsuite,
   python3-gdbm,
   python3-distutils,
   python3-tk,
@@ -41,7 +41,7 @@
   locales-all,
   python3.9-dev,
   python3.9-dbg,
-  libpython3.9-testsuite:native,
+  libpython3.9-testsuite,
   python3-gdbm-dbg,
   gdb,
   python3-distutils,
