--- org/debian/rules.d/scripts/kconfig/Makefile	2020-08-11 14:51:35.530153452 +0000
+++ new/debian/rules.d/scripts/kconfig/Makefile	2020-08-11 14:51:52.622146051 +0000
@@ -2,7 +2,7 @@

 include \\\$(top_rulesdir)/Makefile.inc

-conf: conf.o confdata.o expr.o lexer.lex.o parser.tab.o preprocess.o symbol.o util.o
+conf: conf.o confdata.o expr.o lexer.lex.o parser.tab.o preprocess.o symbol.o

 lexer.lex.o: parser.tab.h

