--- a/debian/linux-cpupower.manpages	2021-07-26 23:18:45.455228357 +0000
+++ b/debian/linux-cpupower.manpages	2021-07-26 23:55:05.175275711 +0000
@@ -1,3 +1,3 @@
 #!/usr/bin/dh-exec
-[i386 amd64 x32] tools/power/x86/turbostat/turbostat.8
-[i386 amd64 x32] tools/power/x86/x86_energy_perf_policy/x86_energy_perf_policy.8
+tools/power/x86/turbostat/turbostat.8
+tools/power/x86/x86_energy_perf_policy/x86_energy_perf_policy.8
--- a/debian/linux-cpupower.install	2021-07-26 23:18:45.455228357 +0000
+++ b/debian/linux-cpupower.install	2021-07-26 23:55:11.539275849 +0000
@@ -1,4 +1,4 @@
 #!/usr/bin/dh-exec
 usr/bin
-[i386 amd64 x32] usr/sbin
+usr/sbin
 usr/share/
