--- a/debian/templates/control.tools-versioned.in	2021-03-19 12:30:17.324928702 +0000
+++ b/debian/templates/control.tools-versioned.in	2021-03-19 12:30:25.892852786 +0000
@@ -24,7 +24,6 @@
  libperl-dev,
  libunwind-dev [amd64 arm64 armel armhf hppa i386 mips mips64 mips64el mipsel powerpc ppc64 ppc64el sh4],
  libopencsd-dev,
- python3-dev,
 Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}, ${python3:Depends}
 Recommends: linux-base
 Provides: linux-tools-@version@
