+++ python3.7-3.7.7/debian/control
@@ -7,7 +7,7 @@ Build-Depends: debhelper (>= 9), dpkg-de
   lsb-release, sharutils,
   libreadline-dev, libncursesw5-dev (>= 5.3), 
   zlib1g-dev, libbz2-dev, liblzma-dev,
-  libgdbm-dev, libdb-dev,
+  libgdbm-dev,
   tk-dev, blt-dev (>= 2.4z), libssl-dev,
   libexpat1-dev, libmpdec-dev (>= 2.4),
   libbluetooth-dev [linux-any] <!pkg.python3.7.nobluetooth>,
+++ python3.7-3.7.7/debian/rules
@@ -354,7 +354,7 @@ common_configure_args = \
 		--prefix=/usr \
 		--enable-ipv6 \
 		--enable-loadable-sqlite-extensions \
-		--with-dbmliborder=bdb:gdbm \
+		--with-dbmliborder=gdbm \
 		--with-computed-gotos \
 		--without-ensurepip \
 		--with-system-expat \
