--- vim-8.2.0716.new/debian/rules	2020-05-31 20:34:36.041213284 +0000
+++ vim-8.2.0716.org/debian/rules	2020-05-31 20:35:19.761371310 +0000
@@ -40,7 +40,6 @@

 OPTFLAGS = --enable-cscope
 ifeq ($(DEB_HOST_ARCH_OS),linux)
-    OPTFLAGS += --enable-canberra
     OPTFLAGS += --enable-gpm
     OPTFLAGS += --enable-selinux --disable-smack
     TINYFLAGS += --enable-selinux --disable-smack
--- vim-8.2.0716.new/debian/control	2020-06-30 14:01:32.079430625 +0000
+++ vim-8.2.0716.old/debian/control	2020-06-30 14:02:02.123539010 +0000
@@ -11,7 +11,6 @@
  cscope <!nocheck>,
  debhelper-compat (= 12),
  libacl1-dev,
- libcanberra-dev,
  libgpmg1-dev [linux-any],
  libgtk-3-dev,
  liblua5.2-dev,
